metriq_gym/metriq_metadata.py

```diff
@@ -23,7 +23,7 @@
     "ibm_nazca": 184,
     "ibm_oslo": 120,
     "ibm_perth": 22,
-    "ibm_prague": 56,   
+    "ibm_prague": 56,
```

Comment:
Completely unrelated to this diff, but since there are edits here it reminded me that this file is here (and it likely should be removed). I would suggest we remove this file entirely. If @cosenal thinks this is outside of scope though, we can mark it as "to-be-removed" in a subsequent MR + issue. Either way.